app/(storyblok)/[[...slug]]/page.tsx

```diff
@@ -49,10 +46,19 @@ storyblokInit({
  * Generate the list of stories to statically render.
  */
 export async function generateStaticParams() {
+  const activeEnv = process.env.NODE_ENV || 'development';
+
+  // Clear out the cached data of this route segment before rebuilding.
+  revalidatePath('/(storyblok)/[[...slug]]/page');
+
+  // Fetch new content from storyblok.
   const storyblokApi: StoryblokClient = getStoryblokApi();
   let sbParams: ISbStoriesParams = {
     version: activeEnv === 'development' ? 'draft' : 'published',
     cv: activeEnv === 'development' ? Date.now() : undefined,
+    resolve_links: '0',
+    resolve_assets: 0,
+    per_page: 100,
```

Comment:
GetAll is supposed to get all the pages but it seemed to be missing the pages from `test/test-stories` without this.